### PR TITLE
Blacklist fetchTokenPrices & add Reactotron

### PR DESCRIPTION
**Pull request by Perronef5**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -38,7 +38,6 @@
   },
   "dependencies": {
     "@apollo/client": "3.5.10",
-    "@coral-xyz/borsh": "^0.26.0",
```
Comment (tyler-whitman) [inline package.json:41]: Dont need this anymore?

File: src/store/store.ts [hunk 2]
```diff
@@ -7,6 +7,12 @@ import reducer from './rootReducer'
 import { walletRestApi } from './slices/walletRestApi'
 import { solanaStatusApi } from './slices/solanaStatusApi'
 import { name as solanaSliceName } from './slices/solanaSlice'
+import Reactotron from '../../ReactotronConfig'
+
+const enhancers = []
+if (Reactotron.createEnhancer) {
```
Comment (tyler-whitman) [inline src/store/store.ts:13]: Should this only happen for DEV?

**Review by tyler-whitman — APPROVED** [review]
I think you accidentally deleted the yarn.lock?

File: package.json (hunk 1 above)
Comment (Perronef5) [reply package.json:41]: Nope

**Comment by Perronef5** [comment]
> I think you accidentally deleted the yarn.lock?

Oops

File: src/store/store.ts (hunk 2 above)
Comment (Perronef5) [reply src/store/store.ts:13]: Yeah let me change that

**Pull request merged**